### ISSUE TITLE
Wrong variable name in pagination

### ISSUE THREAD

**Issue by matlam** [xavierlacot/sfDoctrineRestGeneratorPlugin#40]
In the function queryPagination in data/generator/sfDoctrineRestGenerator/default/parts/queryPagination.php the parameter is called $query, but the body of the function uses $q. This probably breaks pagination and generates a PHP-Notice "Notice: Undefined variable: q" even if pagination is not used.
I think that the bug was introduced by https://github.com/xavierlacot/sfDoctrineRestGeneratorPlugin/commit/6418b75b419f1ec12c39f94cbd9255a1d69dba81

**Comment by nederdirk** [comment]
Yeah, I'm sorry I introduced that bug while refactoring. Since this repository isn't updated as frequently as I wished, I made a hard fork at Recras/sfDoctrineRestGeneratorPlugin . Careful though, we're not trying to be backwards compatible with this repo!
